azure-functions/src/durable/orchestrator_output.rs

```diff
@@ -0,0 +1,56 @@
+use crate::rpc::{typed_data::Data, TypedData};
+use serde_json::Value;
+
+/// # Orchestrator Output
+/// 
+/// Type returned by Orchetrator Functions for Durable Functions
+
+struct OrchestratorOutput(serde_json::Value);
```

Comment:
Let's call this `OrchestrationOutput`, as it's the output of the "orchestration" itself.